docs/templating.md

```diff
@@ -702,13 +702,37 @@ If `tags` was `["food", "beer", "dessert"]`, the above example would output `foo
 
 ## Builtin Filters
 
-Nunjucks has ported most of jinja's filters (click through for documentation):
+Nunjucks has ported most of jinja's filters, and has a few of its own.
+We need to work on our own documentation for filters. Some of them are
+documented below, for the rest you can click through to jinja's site.
+
+### default(value, default, [loose])
+
+(aliased as `d`)
+
+If `value` is `null` or `undefined`, return `default`, otherwise
```

Comment:
`null` and `undefined` usually are treated as the same value. Everywhere we check for `undefined` we also check `null`.

But I see what you mean, this filter is meant to actually only return the default if the variable doesn't even exist in the scope. We can change it to be that.